services/execsrv/exec.go

```diff
@@ -13,11 +13,13 @@ var (
 )
 
 type Store interface {
+	AddSharedVolumes(exec types.Exec) error
 	Create(project string, exec types.Exec) error
 	Get(id string) (types.Exec, error)
 	GetDriver(id string) (string, error)
 	List(filterProject *string, filterProvider *types.Provider, filterActive bool) ([]types.Exec, error)
 	Delete(project, id string) error
+	RemoveSharedVolumes(exec types.Exec) error
```

Comment:
All volumes are shared so I don't think this needs a prefix. I actually think this shouldn't be a new method. This should be done inside the `Create` and `Delete` methods. That's the only time an `ExecVolume` is altered.